hq/app/AppLoader.scala

```diff
@@ -19,6 +19,12 @@ class AppLoader extends ApplicationLoader {
 
     val components = new AppComponents(context)
 
+    components.quartzScheduler.start()
```

Comment:
Should this depend on the environment, like the cache service does? I don't think we want to start this in Test, for example?